src/sentry/models/event.py

```diff
@@ -182,16 +182,20 @@ def get_tag(self, key):
                 return v
         return None
 
+    @property
+    def release(self):
+        return self.get_tag('sentry:release')
+
     @property
     def dist(self):
         return self.get_tag('sentry:dist')
 
     def as_dict(self):
         # We use a OrderedDict to keep elements ordered for a potential JSON serializer
         data = OrderedDict()
-        data['id'] = self.event_id
+        data['event_id'] = self.event_id
         data['project'] = self.project_id
-        data['release'] = self.get_tag('sentry:release')
+        data['release'] = self.release,
```

Comment:
How can this not be a syntax error?

Reply:
Silently creates a tuple. :(

Reply:
Great. @mitsuhiko could you add a test case for these new attributes?

Reply:
it's only used internally, not going to add a test for this.